docs/development-setup.md

```diff
@@ -1,73 +1,164 @@
 # Local Development
 
-It's actually very simple to setup OpenShift locally. We'll use a docker image from Docker Hub. For more OpenShift-focused information, see [this guide](https://docs.openshift.org/latest/getting_started/administrators.html).
 
-First thing to do is to clone this git repo:
+## Running OpenShift
+
+For more options how to install and run OpenShift, please see [this page](https://install.openshift.com/).
+
+We'll use the `oc cluster up` method, since it's very easy to use:
+
+ 1. If you're on Fedora, you can install package origin:
+    ```
+    $ dnf install -y origin
+    ```
+    Now you have command `oc` available on your system
+
+ 2. You can start a single-node cluster like this
+    ```
+    $ oc cluster up
```

Comment:
In addition, the following is required to allow builder ServiceAccount to perform operations from within builder container:
`oc adm policy add-role-to-user admin system:serviceaccount:myproject:builder`

For instance, without this, the plugin `store_metadata_in_osv3` fails with 403 error:  `User "system:serviceaccount:myproject:builder" cannot get builds in project "myproject"`,